cmd/snap/cmd_snap_op.go

```diff
@@ -948,6 +948,47 @@ func (x *cmdRevert) Execute(args []string) error {
 	return nil
 }
 
+var shortSwitchHelp = i18n.G("Switch snap to a different channel")
+var longSwitchHelp = i18n.G(`
+The switch command switch the given snap to a different channel without
+actually doing a refresh.
```

Comment:
The switch command _switches_...  we can also s/actually// below, since it doesn't contribute much.